documentation/leo/03_language.md

```diff
@@ -412,106 +412,48 @@ program test.aleo {
 ```
 
 #### Mapping
-
 The mapping struct allows the programmer to apply updates to a program mapping data structure by calling one of the
 following functions.
 
 #### get
-
 A get command, e.g. `current_value = Mapping::get(counter, addr);`
 Gets the value stored at `addr` in `counter` and stores the result in `current_value`
 If the value at `addr` does not exist, then the program will fail to execute.
 
 #### get_or_init
-
 A get command that initializes the mapping in case of failure, e.g.
 `let current_value: u64 = Mapping::get_or_init(counter, addr, 0u64);`
 Gets the value stored at `addr` in `counter` and stores the result in `current_value`.
 If the key is not present, `0u64` is stored in `counter` and stored in `current_value`.
 
 #### set
-
 A set command, e.g. `Mapping::set(counter, addr, current_value + 1u64);`
 Sets the `addr` entry as `current_value + 1u64` in `counter`.
 
-## Iteration
-
-In the Leo programming language, iteration is supported through the `for` loop construct. This section provides an
-overview of how
-iteration works in Leo and highlights its limitations.
-
-### `for` Loop Syntax
+## For Loops
 
-The syntax for a `for` loop in Leo is as follows:
+For Loops are declared as `for <loop variable> in <lower bound>..<upper bound>`.  Currently, the
```

Comment:
For consistency with the existing docs, let's change <> to {}, so `for {variable} in {lower-bound}..{upper-bound}`. A good example of this to reference is the [mapping type](https://developer.aleo.org/leo/language#mapping).